static-instrumenter/agent-extension/build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+  id("otel.java-conventions")
+  id("com.github.johnrengelman.shadow") version "7.1.2"
+}
+
+description = "Extension for OpenTelemetry Java Agent"
+
+dependencies {
+  annotationProcessor("com.google.auto.service:auto-service")
+  compileOnly("com.google.auto.service:auto-service")
+
+  compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-tooling")
+  compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-instrumentation-api")
+  compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-extension-api")
+  compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
+
+  // TODO: replace with the following file once new javaagent is released
+  compileOnly(files("libs/opentelemetry-muzzle.jar"))
+//  compileOnly("io.opentelemetry.javaagent:opentelemetry-muzzle")
```

Comment:
Tried using SNAPSHOT, seems like the changes are not available yet, but I'll change it as soon as it's possible

Reply:
Hmm, you may need to add the snapshot repo to the gradle config:
```kotlin
repositories {
  mavenCentral()
  maven {
    url = uri("https://oss.sonatype.org/content/repositories/snapshots")
  }
}
```

Reply:
Yeah, I tried it exactly like this and it seemed like the `HelperInjectorListener` wasn't available yet. I see that there is a new version since I've tried so it should be working now

Reply:
Still no luck, gradle seems to always download the 1.13.0-alpha version, so I'll have to look into it a bit more

Reply:
Hmm, did you change the bom version in `dependencyManagement`? Does it still not work?

Reply:
I tried to force using SNAPSHOT only in the necessary dependencies. Setting it directly in `dependencyManagement` finally helped, thanks!

Reply:
It builds locally, but CI seems to not search for it in the right repo

Reply:
Hmm, that's because the jmx-metrics project overrides the repositories (that's one of gradle quirks that I don't really like that much: `repositories {}` completely replaces everything, there doesn't seem to be a simple way of just appending one particular repo for a single project).
Let's just copy the snapshot repo definition over there for now.